### ISSUE TITLE
Dart Sass throws error in pseudo()

### ISSUE THREAD

**Issue by insad** [Simonwep/sassyfication#4]
Dart Sass 1.32.4 when using:

`@include pseudo();`

throws an error:

```
Error in plugin "sass"
Message:
    node_modules\sassyfication\src\lib\_mixins.scss
Error: () isn't a valid CSS value.
  ?
6 │             #{nth($props, $i)}: $val;
  │                                 ^^^^
  ?
  node_modules\sassyfication\src\lib\_mixins.scss 6:33  root stylesheet
```

**Comment by Simonwep** [comment]
Does that work? (never used dart-sass, edit the source file):

```scss
@mixin pseudo($position: absolute, $content: '') {
    @include apply-props(($position, $content), position, content);
}
```

**Comment by insad** [comment]
Yes this works!

**Comment by simonwep** [comment]
Fixed in `v1.0.3` :)
